frontend/src/router/index.js

```diff
@@ -222,6 +210,15 @@ const router = createRouter({
           name: 'manage-facility',
           component: ManageFacilityView,
         },
+        {
+          path: '/manage-users',
```

Comment:
The leading '/' means it won't be nested under /acct-mgmt.

Reply:
fixed.